src/sayt/sayt.ts

```diff
@@ -0,0 +1,20 @@
+/** The name of event fired when SAYT should be hidden. */
+export const SAYT_HIDE = 'sfx::sayt_hide';
+/** The type of the [[SAYT_HIDE]] event payload. */
+export type SaytHidePayload = SaytVisibilityPayload;
+
+/** The name of event fired when SAYT should be hidden. */
+export const SAYT_SHOW = 'sfx::sayt_show';
+/** The type of the [[SAYT_SHOW]] event payload. */
+export type SaytShowPayload = SaytVisibilityPayload;
+
+/**
+ * The base type of the SAYT visibility events.
+ *
+ * @see [[SaytHidePayload]]
+ * @see [[SaytShowPayload]]
+*/
+export interface SaytVisibilityPayload {
```

Comment:
Does this need to be exported?

Reply:
Yeah it does, the `isCorrectSayt` function within sayt.ts in the view layer needs to use this payload interface. Also I think that if I didn't export it typescript would complain because I am already exporting the show and hide interfaces that are referring to visibility interface.